salt/states/zabbix_host.py

```diff
@@ -184,6 +184,10 @@ def _interface_format(interfaces_data):
         inventory = kwargs['inventory']
     if inventory is None:
         inventory = {}
+
+    new_visible_name = kwargs.get('visible_name', '')
+    new_first_name = kwargs.get('first_name', '')
```

Comment:
Please add some documentation for these two new parameters.

Reply:
`visible_name` is documented at the top of the file:
`:param visible_name: Optional - string with visible name of the host, use 'visible_name' instead of 'name' \`
I can comment that this is the mapping for that directive, but it seems a bit redundant...
Will add `firstname` to the documentation of that method as well, however.